lambda/MAS.Tests/Infrastructure/Extensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Shouldly;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace MAS.Tests.Infrastructure
+{
+    public static class Extensions
+    {
+        public static void ShouldMatchApproved(this string content,
+                [System.Runtime.CompilerServices.CallerMemberName] string memberName = null,
```

Comment:
Could we using a using statement for `System.Runtime.CompilerServices.CallerMemberName`